axelar-chains-config/info/stagenet.json

```diff
@@ -2410,6 +2410,8 @@
     "tokenSymbol": "AXL",
     "axelarscanApi": "https://stagenet.api.axelarscan.io",
     "gasPrice": "0.007uaxl",
-    "gasLimit": "auto"
+    "gasLimit": "auto",
+    "depositValue": "100000000",
```

Comment:
Is it not the same as testnet? We can make it the same since stagenet should match testnet